cloud-config/config-build-files/cc-parts/CC_BOOTCMD.yml

```diff
@@ -3,3 +3,4 @@ bootcmd:
 - cloud-init-per once fallocate-swapfile fallocate -l 4G /swapfile
 - cloud-init-per once chmod-swapfile chmod 600 /swapfile
 - cloud-init-per once mkswap-swapfile mkswap /swapfile
+- cloud-init-per once swapon-swapfile swapon /swapfile
```

Comment:
This is the only change I am concerned with.   Possibly if we run some load testing on a cluster while the front end is using a lot of effort to index.   For sure we should ~never put real memory pressure on the front end machines.
I guess we could try simulating running the machine out of RAM?

Reply:
We did do this on v91x0 live or test and did loading testing...It really hard to get it to use swap.  Do you want me to separate out this into its own pr?  Might be better since there is concern about it.

Reply:
I think I would prefer it separate because who knows if it will ever matter but if it does we may need to revert it.

Reply:
I'm just going to switch it out.  'll try to get a load test going tomorrow.  We can always make this change live on v93 or test.

Reply:
The commit would have always been separate since the pr is organized, so revert would have been fine.  I separated it for testing...in case we decided not to merge then we don't have to hold off on the other 5 tickets.